charts/opentelemetry-demo/templates/_objects.tpl

```diff
@@ -82,3 +83,72 @@ spec:
     {{- include "otel-demo.selectorLabels" . | nindent 4 }}
 {{- end}}
 {{- end}}
+
+{{- define "otel-demo.ingress" }}
```

Comment:
I think there is enough logic here to warrant a new CI scenario that tests the chart with ingress enabled.  The collector chart does the same.

Reply:
Added 2 new CI scenarios, and 1 example